test/Spec.hs

```diff
@@ -1217,6 +1217,38 @@ main =
               ]
          in ruleCatchesNot copyFromAnother $ Text.unlines dockerFile
     --
+    describe "multiple COPY to same destination" $ do
+      it "not ok with multiple COPY to same destination - single stage" $
+        let dockerFile =
+              [ "FROM baseimage",
+                "COPY foo /asdf",
```

Comment:
There is a very common pattern that I use often in my docker files:

```
FROM node
COPY package.lock package.json /app/
RUN npm install
COPY src /app
```

Which is a valid pattern as it allows docker to cache more efficiently. I think this rule will flag that incorrectly

Reply:
Good catch. I've played around with different ways to copy directories into docker images and I'm starting to wonder, if I have attempted the impossible with this rule. You see, the destination can be pretty much anything and there is no way to distinguish between files and directories without inspecting the filesystem within the image.

I'm out of ideas here. Maybe this PR should be abandoned.